fstar-helpers/core-models/src/core_arch/x86/interpretations.rs

```diff
@@ -28,18 +36,509 @@ pub mod int_vec {
     }
 
     pub fn _mm256_blend_epi32<const CONTROL: i32>(x: i32x8, y: i32x8) -> i32x8 {
```

Comment:
All of these functions need some comments on what they are doing, not only the code.